src/itest/kotlin/nl/lifely/zac/itest/client/OpenZaakClient.kt

```diff
@@ -0,0 +1,21 @@
+package nl.lifely.zac.itest.client
+
+import com.auth0.jwt.JWT
+import com.auth0.jwt.algorithms.Algorithm.HMAC256
+import nl.lifely.zac.itest.config.ItestConfiguration.OPENZAAK_CLIENT_ID
+import nl.lifely.zac.itest.config.ItestConfiguration.OPENZAAK_CLIENT_SECRET
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_USER_1_ID
+import nl.lifely.zac.itest.config.ItestConfiguration.TEST_USER_1_NAME
+import java.util.Date
+
+object OpenZaakClient {
```

Comment:
No need for an object here since we don't need to keep state or anything right? `generateToken` can be just a plain function in this file.

Reply:
True. 

My idea was to have some hierarchy and encapsulation of the base functionality of the JWT client, and inherit it in OpenZaak and Keycloak. Not too much in common though